### PR TITLE
[BrokenLinksH2] fixing broken link

### PR DESCRIPTION
**Pull request by StephenBrentPeters**

### CODE REVIEW

**Comment by Lauragra** [comment]
@FaithOmbongi , can you please investigate whether we should have a delta method for directory object? We seem to list it in the Methods table but there isn't a link. Thanks.

**Comment by FaithOmbongi** [comment]
> @FaithOmbongi , can you please investigate whether we should have a delta method for directory object? We seem to list it in the Methods table but there isn't a link. Thanks.

Hi @Lauragra, confirmed that **directoryObjects** resource doesn't support the delta function. Only the individual directory objects do and those are already documented.

I'll make a suggestion to @StephenBrentPeters to remove the entry.

File: concepts/delta-query-overview.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ Delta query is currently supported for the following resources. Note that some r
 | Administrative units (preview)                                 | [delta](/graph/api/administrativeunit-delta) function (preview) of the [administrativeUnit](/graph/api/resources/administrativeunit) resource                |
 | Assignment categories                                          | [delta](/graph/api/educationcategory-delta) function of the [educationCategory](/graph/api/resources/educationcategory) resource                                    |
 | Chat messages in a channel                                     | [delta](/graph/api/chatmessage-delta) function (preview) of the [chatMessage](/graph/api/resources/chatmessage)                                              |
-| Directory objects                                              | [delta](/graph/api/directoryobject-delta) function (preview) of the [directoryObject](/graph/api/resources/directoryobject) resource                         |
+| Directory objects                                              | [delta](/graph/api/resources/directoryobject) function (preview) of the [directoryObject](/graph/api/resources/directoryobject) resource                         |
```
Comment (FaithOmbongi) [inline concepts/delta-query-overview.md:112]: Kindly remove this line - The entry is incorrect. I've confirmed from the schema that **directoryObjects** resource doesn't support the delta function. Only the individual directory objects, like users and groups, do.
```suggestion
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @StephenBrentPeters for raising this PR to fix the broken link and flagging this. I've confirmed from the schema that the directoryObjects resource doesn't support the delta function so line 112 in its entirety is a doc error. Kindly pivot your PR to delete the entry. Thank you.

**Comment by FaithOmbongi** [comment]
Close/reopen to retrigger CRLF check + Build.

**Review by FaithOmbongi — APPROVED** [review]
Thank you @StephenBrentPeters for your work on this PR. Please convert the PR from a draft PR to allow your changes to be merged.

**Pull request merged**